node/bft/src/primary.rs

```diff
@@ -717,23 +717,11 @@ impl<N: Network> Primary<N> {
         // Retrieve the batch certificate author.
         let author = certificate.author();
 
-        // Ensure the batch certificate is from the validator.
-        match self.gateway.resolver().get_address(peer_ip) {
```

Comment:
I'm not following why this match case was removed, can you give me an explanation?